worksheet_A/TerminalHacking.cpp

```diff
@@ -41,6 +45,66 @@ int main()
 
 	// TODO: implement the rest of the game
 
+	// initialise variables
+	bool WordInList = false;
+	std::cout << secret;
+	std::string guess;
+	int lives = 4;
+	std::string result = "Lost";
+
+	// main game loop
+	while (lives > 0) 
+	{
+
+		// Ask for user guess
+		std::cout << " what is your guess?" << std::endl;
+		std::cin >> guess;
+
+		// Check to see if guess is in word set
+
+
+
+		for each (std::string word in options)
+		{	
+			if(guess== word){
+				WordInList = true;
+			}
+		}
+
+		// Calculates likeness if guess is valid
+		if (WordInList == true){
+			int likenessScore = likeness(secret, guess);
+			if (likenessScore == wordLength){
+				std::cout << "Gz Noob! You had " << lives << " lives remaining!" << std::endl;
+				result = "Won";
+				lives = 0;
```

Comment:
Needs a comment saying that this causes the loop to end. Or better, use the `break` keyword instead.